crypto/fipsmodule/service_indicator/service_indicator.c

```diff
@@ -7,6 +7,10 @@
 
 #if defined(AWSLC_FIPS)
 
+const char* openssl_version_string(void) {
```

Comment:
Propose re-factoring to a single function that returns a concatenation: `AWSLC_VERSION_TEXT FIPS_SUFFIX`.

You can define `FIPS_SUFFIX` to be empty if `AWSLC_FIPS` is not defined.

Just came up with some quick variable names.

Reply:
I think either way this will end up with about the same level of complexity, but I can make the change since I don't feel super strongly about it anyways